IdentityCore/src/requests/broker/MSIDBrokerKeyProvider.m

```diff
@@ -136,6 +136,36 @@ - (NSData *)brokerKeyWithError:(NSError **)error
     return [self createBrokerKeyWithError:error];
 }
 
+- (NSString *)base64BrokerKeyWithContext:(id<MSIDRequestContext>)context
```

Comment:
should we use this in other places for broker key as well (legacy broker)?

Reply:
done